### PR TITLE
Refactored functionality to classes and added styling

### PR DESCRIPTION
**Pull request by shasherazi**
In this pull request:
- Moved from plain JavaScript objects to JavaScript classes
- Added styling according to the wireframe

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,40 +2,50 @@ const booksList = document.querySelector('.main-books-list-ul');
...
-  bookItem.innerHTML = `
-          <p class="main-books-list-ul-li-title">${title}</p>
-          <p class="main-books-list-ul-li-author">${author}</p>
-          <button class="main-books-list-ul-li-button">Remove</button>
-          <hr class="main-books-list-ul-li-hr">
-      `;
-  booksList.appendChild(bookItem);
+function updateLocalStorage() {
+  // Add item to local storage
+  localStorage.setItem('booksArray', JSON.stringify(Book.books));
```
Comment (Omar-Muhamad) [inline src/index.js:42]: - [ ] When I remove a book from the list of books it remains on the local storage and when I reload the page It returns again to appear on the page, So please fix that.

chrome-capture-2023-0-10

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @shasherazi and @Mekuseo,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (shasherazi) [reply src/index.js:42]: I don't know why it's happening becuase https://github.com/shasherazi/awesomeBooks/blob/c6de2aae34754d55f1dbf46c4a22e7496c15164d/src/index.js#L66 should update the `localStorage` after removing the book. And it is doing so in mine and my partner's browser. Still I moved the updating logic in the `remove()` function itself @Omar-Muhamad

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @shasherazi and @Mekuseo, 

To highlight 👍🏽 
✔️ Your app works as expected
✔️ There are no linter errors

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**